objectherkenning_openbare_ruimte/data_sampling/decos_helper.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import pathlib
+
+import geopandas as gpd
+import pandas as pd
+
+RD_CRS = "EPSG:28992"  # CRS code for the Dutch Rijksdriehoek coordinate system
```

Comment:
I don't like this global value being here and repeated across multiple files.
In case of a need for update, it's easy to forget one.
Maybe you can add them to the`__init__` file?

Reply:
The CRS code is fixed by international standards, I don't think this will ever change.